ext/FluxnetSimulations.jl

```diff
@@ -0,0 +1,308 @@
+        not_missing_mask = .~ismissing.(atmos_precip_data)
+        P_liq_data = -1 .* Float64.(atmos_precip_data[not_missing_mask])
+        P_liq = TimeVaryingInput(
+            seconds_since_start_date[not_missing_mask],
+            P_liq_data ./ 1000 ./ DATA_DT,
+        )
+        P_snow = TimeVaryingInput(
+            seconds_since_start_date[not_missing_mask],
+            zeros(length(seconds_since_start_date[not_missing_mask])),
+        )
+    end
+
+    # Construct the drivers. For the start date we will use the UTC time at the
+    # start of the simulation
+    atmos = ClimaLand.PrescribedAtmosphere(
+        P_liq,
+        P_snow,
+        atmos_T,
+        atmos_u,
+        atmos_q,
+        atmos_P,
+        start_date,
+        atmos_h,
+        earth_param_set;
+        c_co2,
+    )
+
+    zenith_angle =
+        (t, s) -> default_zenith_angle(
+            t,
+            s;
+            insol_params = earth_param_set.insol_params,
+            longitude = long,
+            latitude = lat,
+        )
+    radiation = ClimaLand.PrescribedRadiativeFluxes(
+        FT,
+        SW_d,
+        LW_d,
+        start_date,
+        θs = zenith_angle,
+        earth_param_set = earth_param_set,
+    )
+
+    MODIS_LAI_path = ClimaLand.Artifacts.get_modis_lai_fluxnet_data(site_ID)
+    MODIS_LAI, header = readdlm(MODIS_LAI_path, ',', header = true)
+    dates = DateTime.(MODIS_LAI[:, 1])
+
+    LAI_dt = Second(dates[2] - dates[1]).value
+    LAI_seconds_since_start_date =
+        [Second(date - start_date).value for date in dates]
+    LAI = TimeVaryingInput(LAI_seconds_since_start_date, FT.(MODIS_LAI[:, 2]))
+    maxLAI = FT(maximum(MODIS_LAI[:, 2]))
+
+    return (; atmos, radiation, LAI, maxLAI)
+end
+
+"""
+    snow_precip_fraction(air_temp, hum)
+
+Estimate the fraction of precipitation that is in snow form,
+given the air temperature at the surface in K and the relative humidity
+(between 0 and 1).
+
+See Jennings, K.S., Winchell, T.S., Livneh, B. et al.
+Spatial variation of the rain–snow temperature threshold across the
+Northern Hemisphere. Nat Commun 9, 1148 (2018).
+https://doi.org/10.1038/s41467-018-03629-7
+"""
+function snow_precip_fraction(air_temp, hum)
+    air_temp_C = air_temp - 273.15
+    α = -10.04
+    β = 1.41
+    γ = 0.09
+    snow_frac = (1.0 / (1.0 + exp(α + β * air_temp_C + γ * hum)))
+    return snow_frac
+end
+
+function set_fluxnet_ic!(
+    Y,
+    site_ID,
+    start_date,
+    hour_offset_from_UTC,
+    model::LandModel,
+)
+    fluxnet_csv_path = ClimaLand.Artifacts.experiment_fluxnet_data_path(site_ID)
+    driver_data = readdlm(fluxnet_csv_path, ',')
+    columns = driver_data[1, :]
+    labels = ("SWC_F_MDS_1", "TS_F_MDS_1", "TA_F")
+    indices = [findfirst(columns .== label) for label in labels]
+    index_map = Dict(zip(labels, indices))
+
+    LOCAL_DATETIME = DateTime.(format.(driver_data[2:end, 1]), "yyyymmddHHMM")
+    UTC_DATETIME = LOCAL_DATETIME .+ Dates.Hour(hour_offset_from_UTC)
```

Comment:
```suggestion
    local_datetime = DateTime.(format.(driver_data[2:end, 1]), "yyyymmddHHMM")
    utc_datetime = local_datetime .+ Dates.Hour(hour_offset_from_UTC)
```
we should use the normal naming convention since these arent `const`s